### ISSUE TITLE
Error in national holidays

### ISSUE THREAD

**Issue by GetxoSystems** [xmera-circle/redmine_workload#33]
### Contact Details

jortizdezarate@getxo.eus

### What did you expect?

I will appreciate If the issue is resolved, I diagnosed the issue and found the error but I lack the knowledge in ruby/rails

### What has happend instead?

The error is related with postgres database, so only affect to redmine installs using postgres.
The table  wl_national_holidays contains 2 fields start and end, when quering the national holidays
SELECT * FROM public.wl_national_holidays WHERE (start >= '2024-01-01') AND (end >= '2024-01-01')
postgres parser thinks the reserver word end is the end of a transaction, so it gives error, and the web interface returns an internal error
if I query
SELECT * FROM public.wl_national_holidays WHERE (public.wl_national_holidays.end >= '2024-01-01')  the query works flawlessly
So  I suggest to rename the name of end field in table wl_national_holidays

### What could be a possible solution?

The solution is written in previous field

### How can we reproduce the problem?

install redmine with postgres database. 

### In what environment are you running the plugin?

Our installation is a debian 12, redmine 5.1 and postgres 16, working correctly with many plugins installed
no containers and all installed in a single production virtual machine over Vmware 7.1

### Version

1.0.0

### Relevant log output

```shell
F, [2024-01-02T11:37:20.196942 #1435] FATAL -- :   
ActiveRecord::StatementInvalid (PG::SyntaxError: ERROR:  error de sintaxis en o cerca de «end»
LINE 1: ...onal_holidays" WHERE (start <= '2023-12-23') AND (end >= '20...
                                                             ^
):
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by GetxoSystems** [comment]
The web server in nginx

**Comment by liaham** [comment]
Hi  @GetxoSystems ,

Thank you very much for your bug report! We appreciate any help.

What you describe is a solved issue #16.

Please install the latest release of the plugin: https://github.com/xmera-circle/redmine_workload/releases/tag/3.0.2.

Best Regards,

@liaham

**Comment by GetxoSystems** [comment]
Liane,
Attached you can find the translation to spanish (spain) of redmine workload plugin
Thanks for your fast reply




Julio Ortiz de Zarate Crespo
Telekomunikazio- Ingeniaria / Ingeniero/a de Telecomunicación
Gobernu Ireki eta Modernizazio Arloa / Área de Gobierno Abierto y Modernización

[cid:Escudo_7cbd05f0-d588-4757-9edb-9bd982e4fe2c.jpg]
[cid:bizi_7737e9f3-4d71-4a50-bdf8-7251a76e3760.jpg].    .

✆ +34944660218 / 3450<tel:3450>
✉ ***@***.******@***.***> Ⓦ www.getxo.eus<https://www.getxo.eus/>
[cid:ubicacion2_28d46b49-206c-4d8c-b19d-4ef2fe4c4a86.png] Basagoiti Etorbidea, 20 · 48992 · Getxo · Bizkaia


[cid:036_sm_fb_a1347472-45be-4a0b-af83-b901fa688c57.png]<https://www.facebook.com/GetxoUdala/>  [cid:logo-black_71f100eb-1275-4930-84b5-c65aa15e57ec.png] <https://twitter.com/getxoudala>   [cid:036_sm_y2_45d41b4c-7f16-42de-b619-b5cde39239df.png] <https://www.youtube.com/channel/UChEUKdARrLu5ylXvJFUrccQ>   <https://www.instagram.com/user_name_here> .
.

De: Liane Hampe ***@***.***
Enviado el: martes, 2 de enero de 2024 12:24
Para: xmera-circle/redmine_workload
CC: Julio Ortiz de Zarate Crespo; Mention
Asunto: Re: [xmera-circle/redmine_workload] Error in national holidays (Issue #33)


Hi @GetxoSystems<https://github.com/GetxoSystems> ,

Thank you very much for your bug report! We appreciate any help.

What you describe is a solved issue #16<https://github.com/xmera-circle/redmine_workload/issues/16>.

Please install the latest release of the plugin: https://github.com/xmera-circle/redmine_workload/releases/tag/3.0.2.

Best Regards,

@liaham<https://github.com/liaham>

**Comment by mboremski** [comment]
Dear @GetxoSystems,
unfortunately, attachments from Github are not added to the issue if they are sent via email as a reply.

We can include your translation into the next release.
Would you be kind enough to create a new issue for this and paste the file there?
Thanks alot for your support!
